tests/unit/test_kwargs/test_validations.py

```diff
@@ -58,13 +57,10 @@ def test_raises_when_reserved_kwargs_are_misused(reserved_kwarg: str) -> None:
     with pytest.raises(ImproperlyConfiguredException):
         Litestar(route_handlers=[handler_with_dependency])
 
-    # these kwargs are set to Any when the signature model is generated,
-    # because pydantic can't handle generics for non pydantic classes. So these tests won't work for aliased parameters.
-    if reserved_kwarg not in SKIP_VALIDATION_NAMES:
```

Comment:
This restriction no longer exists, since we moved away from the Pydantic signature model.